ShareMenu/ShareViewController.swift

```diff
@@ -17,9 +17,9 @@ class ShareViewController: UIViewController {
         static let dummyError: Dismiss = .init()
     }
     
-    @IBOutlet weak var qrImageView: UIImageView!
-    @IBOutlet weak var textLabel: UILabel!
-    @IBOutlet weak var sharingItemSwitch: UISegmentedControl!
+    @IBOutlet private weak var qrImageView: UIImageView!
+    @IBOutlet private weak var textLabel: UILabel!
+    @IBOutlet private weak var sharingItemSwitch: UISegmentedControl!
     
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: ShareMenu/ShareViewController.swift;
  Line: 23;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)